IQSS/dataverse-frontend
# 311 - Integrate breadcrumbs with owner information

## What this PR does / why we need it:
Display the actual breadcrumb routes from the new owner info recently added to the Dataverse API

This issue depends on:
- https://github.com/IQSS/dataverse-client-javascript/issues/122
- https://github.com/IQSS/dataverse-client-javascript/issues/121

## Which issue(s) this PR closes:

- Closes #311 

## Special notes for your reviewer:
I changed the Home page to be the collection page since now we're able to display datasets from collections different than the root one.

The e2e tests are failing because the GitHub Action is using an unstable image of Dataverse instead of the image containing the necessary Dataverse changes. The Dataverse branch should be merged prior to this one

## Suggestions on how to test this:
### Step 1: Run the development environment
1. Run `npm I`
3. `cd packages/design-system && npm run build`
4. `cd ../../`
5. Check that you have a  `.env` file such as the .env.example, with the `VITE_DATAVERSE_BACKEND_URL=http://localhost:8000` variable
6.  `cd dev-env`
7. `./run-env.sh unstable`
8. To check the environment go to <http://localhost:8000> and check your local dataverse installation

### Step 2: Test breadcrumbs
1. Go to <http://localhost:8000>
2. Login as admin using username: dataverseAdmin and password: admin1
3. Create a new Dataverse collection
4. Create a new Dataset
5. Upload a file to the dataset
7. Go to <http://localhost:8000/spa> and select your dataset in the list
8. From the list of files in the Dataset page select your file to go to the File page
9. Now from the File page check the breadcrumbs are being displayed correctly and click the corresponding dataset breadcrumb
10. Now you should be in the Dataset Page. Look at the breadcrumbs and click the collection link
11. Now you should be in the Collection view. Now click on the Root breadcrumb to go to the initial page

## Does this PR introduce a user interface change? If mockups are available, please link/include them here:
No

## Is there a release notes update needed for this change?:
Add breadcrumbs navigation to the SPA

## Additional documentation:
